### PR TITLE
added telegram module

### PR DESCRIPTION
**Pull request by denyskulykov**

### CODE REVIEW

File: cvs_gen.py [hunk 1]
```diff
@@ -1,19 +1,24 @@
 import csv
-
 from database import view_data, get_all
+from helpers import number_format
+
+headers = view_data + ('2-th ph', '3-th ph',)
 
 path_output_file = "report.csv"
 
 with open(path_output_file, 'w') as csv_file:
```
Comment (stasya72008) [inline cvs_gen.py]: Please, specify encoding

File: cvs_gen.py [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
     writer.writeheader()
     lines = get_all()
-    for _line in lines:
-        writer.writerow(dict(zip(view_data, _line)))
 
+    for __line in lines:
+        _line = list(__line)
+        numbers = _line[0].split('\n')
+        if len(numbers) != 1:
+            _line[0] = numbers[0]
```
Comment (stasya72008) [inline cvs_gen.py]: Redundant line

File: cvs_gen.py [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
+            [_line.append(number_format(item)) for item in numbers[1:]]
+
+        _line[0] = number_format(_line[0])
```
Comment (stasya72008) [inline cvs_gen.py]: Please. move this line before if condition

File: database.py [hunk 4]
```diff
@@ -83,17 +93,49 @@ def is_not_phone_exists(phone):
...
+        Name VARCHAR(100) NOT NULL,
+        Seen VARCHAR(30),
+        Profile VARCHAR(255),
+        CONSTRAINT unique_local UNIQUE (Number)
+        );
+    """
+    _exe_raw_sql(sql)
+
+
+# ToDo Drop hardcore
```
Comment (stasya72008) [inline database.py]: It's hardcoDe :P

File: database.py [hunk 5]
```diff
@@ -83,17 +93,49 @@ def is_not_phone_exists(phone):
...
+def insert_into_telegram(number, name, _id=1, seen=None, profile=None):
```
Comment (stasya72008) [inline database.py:112]: Maybe you could reuse existing method for insertion?

File: cvs_gen.py [hunk 6]
```diff
@@ -1,19 +1,24 @@
 import csv
```
Comment (stasya72008) [inline cvs_gen.py]: Please, rename this file to csv_gen :)

File: cvs_gen.py [hunk 7]
```diff
@@ -1,19 +1,24 @@
...
 
 path_output_file = "report.csv"
 
 with open(path_output_file, 'w') as csv_file:
-    writer = csv.DictWriter(csv_file, fieldnames=view_data,
-                            delimiter=';', lineterminator='\n')
+    writer = csv.DictWriter(
+        csv_file, fieldnames=headers, delimiter=',', lineterminator='\n')
     writer.writeheader()
     lines = get_all()
```
Comment (stasya72008) [inline cvs_gen.py]: You can drop this line and use for __line in get_all(), like in telegram reports

File: telegramm.py [hunk 8]
```diff
@@ -0,0 +1,125 @@
+import getpass
```
Comment (stasya72008) [inline telegramm.py:1]: Unused import

File: telegramm.py [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+chrome_options.add_argument("--disable-dev-shm-usage")
+chrome_options.add_argument("--disable-gpu")  # applicable to windows os only
+chrome_options.add_argument('--no-sandbox')
+
+driver = webdriver.Chrome(chrome_options=chrome_options)
+driver.maximize_window()
+
+wait = WebDriverWait(driver, 10)
+
+lines = get_all()[:]
```
Comment (stasya72008) [inline telegramm.py]: :)

File: telegramm.py [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+        if is_telegram_acount(number):
+            continue
+
+        try:
+            # Addition of phone number
+            print('-' * 50)
+            print(number)
+
+            driver.find_element_by_xpath('//*[@class="tg_head_btn dropdown-toggle"]').click()
```
Comment (stasya72008) [inline telegramm.py]: How do you think, for whom constants file was created as a good coding manner?)

File: telegramm.py [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+            actions.click()
+            actions.send_keys(number)
+            actions.perform()
+            actions.reset_actions()
+
+            driver.find_element_by_xpath('//*[contains(text(),"Save")]').click()
+
+            # Error Verification
+            try:
+                wait_err = WebDriverWait(driver, 3)
```
Comment (stasya72008) [inline telegramm.py:56]: Is using of implicit waits really necessary when you have an explicit waiter right after it?

File: database.py (hunk 4 above)
Comment (denyskulykov) [reply database.py]: :P

File: database.py (hunk 5 above)
Comment (denyskulykov) [reply database.py:112]: Nope, we have the parameters logic here.
It's separate on both tables.

File: telegramm.py (hunk 8 above)
Comment (denyskulykov) [reply telegramm.py:1]: Now It's used )

File: telegramm.py (hunk 11 above)
Comment (denyskulykov) [reply telegramm.py:56]: have discussed

**Pull request merged**